scss/_focus.scss

```
@@ -55,16 +55,21 @@ a {
 .custom-select,
 .custom-file .custom-file-label,
 .nav-link,
-.btn {
+.btn, 
+.official-banner .btn.expander {
   &:focus {
     @include focus-default;
   }
 }
+.official-banner .btn.expander {
+  &:focus {
+    outline-color: $white!important;
```

```suggestion
    outline-color: $white;
```
This important flag is not needed. The outline selector is after the default given above.